modules/auto-update/auto-updater.js

```diff
@@ -0,0 +1,240 @@
+
+const path = require('path');
```

Comment:
Plase update the folder structure like it's stated in architecture document:
Module should look like:
modules
   auto-updater
           auto-updater-module-interface.js
           implementation
                   ot-auto-updater.js
                   
 auto-updater-module-interface should behave something like proxy between the node and implementation. So when calling methods from auto updater we should always call interface